utils/test/TestKickStart.bat

```diff
@@ -0,0 +1,4 @@
+
+@ECHO ON
+python TestRunner.py
```

Comment:
That's right, I removed it because it wasn't working for me. I'll add it back in and see if I can find out what was happening.